### PR TITLE
Bug 1441822 Arranged info on optimized install

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1441822

cc @sferich888 @jeremyeder

### CODE REVIEW

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -60,6 +60,11 @@ xref:../../install_config/install/rpm_vs_containerized.adoc#install-config-insta
 Containerized Hosts] to ensure that you understand the differences between these
 methods, then return to this topic to continue.
 
+If you are interested in optimizing your installation, including recommendations
```
Comment (jeremyeder) [inline install_config/install/advanced_install.adoc]: Perhaps change this wording to indicate what might constitute "interest" by saying

For larger scale installs and optimizing install time, see the ...

Perhaps drop the "recommendations for networking", not sure what that's referring to?

Comment (sferich888) [inline install_config/install/advanced_install.adoc]: +1

Comment (bfallonf) [inline install_config/install/advanced_install.adoc]: @jeremyeder Sure. Changed to:

"For large-scale installs, including suggestions for optimizing install time,
see the Scaling and Performance Guide."

Let me know if there's anything else.

**Comment by bfallonf** [comment]
@ahardin-rh @adellape @mburke5678 Should be ok for a peer review. Thanks in advance!

**Comment by mburke5678** [comment]
LGTM

**Comment by bfallonf** [comment]
Thanks @mburke5678 . I'll merge.

**Comment by bfallonf** [comment]
I don't think a rev history is needed.

**Pull request merged**